### PR TITLE
dev-lua/luaposix: new version 35.0

### PR DESCRIPTION
**Pull request by winterheart**
New version, new maintainer, added dependency dev-lua/lua-bit32 (#603812)
Closes: https://bugs.gentoo.org/603812
Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: Azamat H. Hackimov <azamat.hackimov@gmail.com>

### CODE REVIEW

File: dev-lua/lua-bit32/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<name>Azamat H. Hackimov</name>
+</maintainer>
+<maintainer type="project">
+<email>proxy-maint@gentoo.org</email>
+<name>Proxy Maintainers</name>
+</maintainer>
+<upstream>
+<remote-id type="github">keplerproject/lua-compat-5.3</remote-id>
+</upstream>
+</pkgmetadata>
```
Comment (juippis) [inline dev-lua/lua-bit32/metadata.xml:15]: Could you indent these better? 
https://github.com/gentoo/gentoo/blob/master/skel.metadata.xml

File: dev-lua/luaposix/luaposix-35.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+RDEPEND="dev-lang/lua:0=
+	dev-lua/lua-bit32"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_compile() {
+	./build-aux/luke package="${PN}" version="${PV}" \
```
Comment (juippis) [inline dev-lua/luaposix/luaposix-35.0.ebuild:22]: Needs to `|| die`.

File: dev-lua/luaposix/luaposix-35.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		PREFIX="${ED}/usr" \
+		INST_LIBDIR="${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_CMOD lua)" \
+		INST_LUADIR="${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_LMOD lua)" \
+		CFLAGS="${CFLAGS}" CC="$(tc-getCC)"
+}
+
+src_install() {
+	./build-aux/luke install \
```
Comment (juippis) [inline dev-lua/luaposix/luaposix-35.0.ebuild:30]: Here too.

File: dev-lua/luaposix/metadata.xml [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+<email>azamat.hackimov@gmail.com</email>
+<name>Azamat H. Hackimov</name>
+</maintainer>
+<maintainer type="project">
+<email>proxy-maint@gentoo.org</email>
+<name>Proxy Maintainers</name>
+</maintainer>
+<upstream>
+<remote-id type="github">luaposix/luaposix</remote-id>
+</upstream>
```
Comment (juippis) [inline dev-lua/luaposix/metadata.xml:14]: Here too.

**Comment by winterheart** [comment]
@juippis done

**Review by juippis — APPROVED** [review]
```
 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib64/lua/5.1/posix/ctype.so
 * /usr/lib64/lua/5.1/posix/dirent.so
 * /usr/lib64/lua/5.1/posix/errno.so
 * /usr/lib64/lua/5.1/posix/fcntl.so
 * /usr/lib64/lua/5.1/posix/fnmatch.so
 * /usr/lib64/lua/5.1/posix/glob.so
 * /usr/lib64/lua/5.1/posix/grp.so
 * /usr/lib64/lua/5.1/posix/libgen.so
 * /usr/lib64/lua/5.1/posix/poll.so
 * /usr/lib64/lua/5.1/posix/pwd.so
 * /usr/lib64/lua/5.1/posix/sched.so
 * /usr/lib64/lua/5.1/posix/signal.so
 * /usr/lib64/lua/5.1/posix/stdio.so
 * /usr/lib64/lua/5.1/posix/stdlib.so
 * /usr/lib64/lua/5.1/posix/sys/msg.so
 * /usr/lib64/lua/5.1/posix/sys/resource.so
 * /usr/lib64/lua/5.1/posix/sys/socket.so
 * /usr/lib64/lua/5.1/posix/sys/stat.so
 * /usr/lib64/lua/5.1/posix/sys/statvfs.so
 * /usr/lib64/lua/5.1/posix/sys/time.so
 * /usr/lib64/lua/5.1/posix/sys/times.so
 * /usr/lib64/lua/5.1/posix/sys/utsname.so
 * /usr/lib64/lua/5.1/posix/sys/wait.so
 * /usr/lib64/lua/5.1/posix/syslog.so
 * /usr/lib64/lua/5.1/posix/termio.so
 * /usr/lib64/lua/5.1/posix/time.so
 * /usr/lib64/lua/5.1/posix/unistd.so
 * /usr/lib64/lua/5.1/posix/utime.so
>>> Auto-cleaning packages...
```

**Pull request closed without merging**